lib/ae_mdw/collection.ex

```diff
@@ -136,17 +136,21 @@ defmodule AeMdw.Collection do
   def integer_256bit(), do: :integer_256bit
   @spec binary() :: :binary
   def binary(), do: :binary
+  @spec interval_by() :: :interval_by
+  def interval_by(), do: :interval_by
 
   defp get_min_key(:pos_integer), do: 0
   defp get_min_key(:integer), do: Util.min_int()
   defp get_min_key(:integer_256bit), do: Util.min_256bit_int()
   defp get_min_key(:binary), do: Util.min_bin()
+  defp get_min_key(:interval_by), do: :day
   defp get_min_key(x), do: x
 
   defp get_max_key(:pos_integer), do: Util.max_int()
   defp get_max_key(:integer), do: Util.max_int()
   defp get_max_key(:integer_256bit), do: Util.max_int()
   defp get_max_key(:binary), do: Util.max_256bit_bin()
+  defp get_max_key(:interval_by), do: :week
```

Comment:
This is never used and doesn't really make sense. We never want to get a range of values that includes week and day. `:day < :week < :month` is not true either

Reply:
It is used in the tests here: https://github.com/aeternity/ae_mdw/pull/1921/files#diff-a571664afdbaba1cd53426420278d7439b14675881d08b82f57454b8a2e070fcR800 basically for getting all the related statistics. But yes, it probably won't be used outside of the tests, so I could write them out there without helpers (but as a side not, I made the last value `:week`, so it works, e.g. `:day < :month < :week`).